### PR TITLE
My task form

### PR DESCRIPTION
**Pull request by Mohamed22Yousri**

### CODE REVIEW

File: src/modules/tasks/components/TaskForm/FormAddAndEdit.tsx [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+      
+    }
+  }
+  const getprojects = async () => {
+    const res = await axiosInstance.get(PROJECT_URL.GET_PROJECTS)
+    console.log(res.data.data);
+    
+    setProjects(res.data.data)
+  }
+  const getUsers = async () => {
```
Comment (Ahmed-Abou-Emran) [inline src/modules/tasks/components/TaskForm/FormAddAndEdit.tsx:65]: We will need to set a large number like 1000, and make the select for the users searchable to be able to choose the employee we want. The current implementation will show only 5.

Comment (Ahmed-Abou-Emran) [reply src/modules/tasks/components/TaskForm/FormAddAndEdit.tsx:65]: you can use this package for searchable select.
https://react-select.com/

File: src/modules/tasks/components/TaskForm/HeaderForm.tsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
+import styles from "./CSS/TaskForm.module.css";
+import { Link } from "react-router-dom";
+const HeaderForm = () => {
+  return (
+    <>
+      <div className="bg-white">
+        <div className="container">
+          <div className={styles["title"]}>
+            <svg
```
Comment (Ahmed-Abou-Emran) [inline src/modules/tasks/components/TaskForm/HeaderForm.tsx]: We can extract this svg as it's own component.

**Comment by Mohamed22Yousri** [comment]
تمام كدا عدلتهم

**Review by Ahmed-Abou-Emran — APPROVED** [review]

**Pull request merged**